js/interface-lists.js

```diff
@@ -462,7 +490,7 @@ function attahObservers() {
           var values = [];
 
           values.push({
-            value: widgetData.pollColumn,
+            value: widgetData.surveyColumn,
```

Comment:
Why is this changed?

Reply:
This is needed to avoid an error like this.
![can-save-without-a-field](https://user-images.githubusercontent.com/53430352/64613332-8fc89b00-d3de-11e9-86eb-0342a5ae33eb.gif)

As you may see we able to save surveys without selecting any survey field. Because we were checking only the poll field.